### PR TITLE
Fix mutation modal confirm

### PR DESCRIPTION
**Pull request by bprize15**
resolves https://github.com/oncokb/oncokb-pipeline/issues/316

### CODE REVIEW

File: src/main/webapp/app/shared/modal/AddMutationModal.tsx [hunk 1]
```diff
@@ -359,8 +361,9 @@ function AddMutationModal({
   }
 
   const fetchNormalAlterationDebounced = useCallback(
-    _.debounce((newAlteration: string, alterationIndex: number, alterationData: AlterationData[]) => {
-      fetchNormalAlteration(newAlteration, alterationIndex, alterationData);
+    _.debounce(async (newAlteration: string, alterationIndex: number, alterationData: AlterationData[]) => {
+      await fetchNormalAlteration(newAlteration, alterationIndex, alterationData);
```
Comment (calvinlu3) [inline src/main/webapp/app/shared/modal/AddMutationModal.tsx:365]: We should probably handle scenario when the async function fails. I can create a ticket to make sure that all async function errors are caught.

File: src/main/webapp/app/shared/modal/AddMutationModal.tsx [hunk 2]
```diff
@@ -1009,13 +1024,19 @@ interface IAddMutationModalFieldProps {
...
   return (
     <Row className="align-items-center mb-3">
       <Col className="px-0 col-3 mr-3">
-        <span>{label}</span>
+        <span className="mr-2">{label}</span>
+        {isLoading && (
+          <span>
+            <Spinner size="sm" />
+          </span>
+        )}
```
Comment (calvinlu3) [inline src/main/webapp/app/shared/modal/AddMutationModal.tsx:1039]: Could just be me, but the spinner position seems slightly below the center.

Comment (calvinlu3) [reply src/main/webapp/app/shared/modal/AddMutationModal.tsx:1039]: Also, what do you think about using $oncokb-blue for spinner color (same as LoadingIndicator).

Comment (calvinlu3) [reply src/main/webapp/app/shared/modal/AddMutationModal.tsx:1039]: image

File: src/main/webapp/app/shared/modal/AddMutationModal.tsx (hunk 1 above)
Comment (bprize15) [reply src/main/webapp/app/shared/modal/AddMutationModal.tsx:365]: Ok sounds good, we can come back through for more error checking.

**Review by calvinlu3 — APPROVED** [review]
LGTM

**Pull request merged**